bin/composer

```diff
@@ -12,4 +13,4 @@ docker run \
    -v ${PROJECTDIR}/vendor:/app/vendor \
    -v ${PROJECTDIR}/scripts:/app/scripts \
    -v composer-cache:/tmp/cache \
-   mobomo/composer $ARGS
+   ${IMAGE_MAINTAINER}/composer $ARGS
```

Comment:
This doesn't make sense to me. I agree it shouldn't be mobomo, (we need to create a docker hub org to represent drupalstand), but if we pull it from a variable, this could change. We edit the environment file, set it to our org, then this line breaks because this is docker hub organization. We shouldn't have a dependency on a docker hub org with a composer image existing.